pytorch_vision_once_for_all.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: Once for All: Train One Network and Specialize it for Efficient Deployment
+summary: We propose to train a once-for-all (OFA) network that supports diverse architectural settings by decoupling training and search, thus achieving efficient inference across many devices and resource constraints, especially on edge devices.
```

Comment:
Could it be much shorter? Like 10-15 words max?

Reply:
Sure, how about current version?